{
  "uid" : "90fa97df71bcde79",
  "name" : "with short signature",
  "fullName" : "Contract: ECDSA recover with invalid signature.with short signature",
  "historyId" : "2d3465e3c61c9a0c3c91d93d02c7f5b9",
  "time" : {
    "start" : 1635782756700,
    "stop" : 1635782757654,
    "duration" : 954
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "broken",
  "statusMessage" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067edc0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
  "statusTrace" : "Error: Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067edc0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))\n                    at Context.<anonymous> (test/utils/cryptography/ECDSA.test.js:58:37)\n                    at processImmediate (node:internal/timers:464:21)",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Contract: ECDSA recover with invalid signature"
  }, {
    "name" : "testClass",
    "value" : "Contract: ECDSA recover with invalid signature"
  }, {
    "name" : "testMethod",
    "value" : "with short signature"
  }, {
    "name" : "package",
    "value" : "Contract: ECDSA recover with invalid signature"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "46ebe4fc6a011ea4",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0433160>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782648489,
        "stop" : 1635782649051,
        "duration" : 562
      }
    }, {
      "uid" : "55b7839075f9aad6",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a07f5fd0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782647926,
        "stop" : 1635782648491,
        "duration" : 565
      }
    }, {
      "uid" : "1f812ef0765a9061",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a044bcd0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782627199,
        "stop" : 1635782627867,
        "duration" : 668
      }
    }, {
      "uid" : "5b48f1008dd68ba7",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a04182b0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782617489,
        "stop" : 1635782618453,
        "duration" : 964
      }
    }, {
      "uid" : "ce4d6e0638c78c3c",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0420970>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782599260,
        "stop" : 1635782599823,
        "duration" : 563
      }
    }, {
      "uid" : "acf929cfb4055a31",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0437d60>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782544114,
        "stop" : 1635782544682,
        "duration" : 568
      }
    }, {
      "uid" : "4673205134899c02",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f238be8c100>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782542432,
        "stop" : 1635782542993,
        "duration" : 561
      }
    }, {
      "uid" : "185f696229221677",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0442130>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782533012,
        "stop" : 1635782533592,
        "duration" : 580
      }
    } ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 4,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 4
      },
      "items" : [ {
        "uid" : "c58ac274651c60e",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/8//#testresult/c58ac274651c60e",
        "status" : "broken",
        "statusDetails" : "Returned error: Transaction simulation failed: Error processing Instruction 0: invalid account data for instruction",
        "time" : {
          "start" : 1635590600188,
          "stop" : 1635590612657,
          "duration" : 12469
        }
      }, {
        "uid" : "d4a40fdca93075ef",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/7//#testresult/d4a40fdca93075ef",
        "status" : "broken",
        "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0437100>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
        "time" : {
          "start" : 1635555035935,
          "stop" : 1635555036916,
          "duration" : 981
        }
      }, {
        "uid" : "5bd3e4a080250649",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/6//#testresult/5bd3e4a080250649",
        "status" : "broken",
        "statusDetails" : "HH110: Invalid JSON-RPC response received: <html>\n                <head><title>502 Bad Gateway</title></head>\n                <body>\n                <center><h1>502 Bad Gateway</h1></center>\n                <hr><center>nginx/1.18.0 (Ubuntu)</center>\n                </body>\n                </html>\n                ",
        "time" : {
          "start" : 1635551257351,
          "stop" : 1635551257414,
          "duration" : 63
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "90fa97df71bcde79.json",
  "parameterValues" : [ ]
}